chore(docs) minor refresh on math skills test page (#22029)
mdn/content: files/en-us/learn/javascript/first_steps/test_your_skills_colon__math/index.md

{{learnsidebar}}

The aim of the tests on this page is to assess whether you've understood the [Basic math in JavaScript — numbers and operators](/en-US/docs/Learn/JavaScript/First_steps/Math) article.

> > **Note:** If there is an error in your code it will be logged into the results panel on this page (or in the browser's JavaScript console in the downloadable version) to help you.

Reviewer: ```suggestion
> **Note:** If there is an error in your code, it will be logged into the results panel on this page (or in the browser's JavaScript console for the downloadable version) to help you.
```

> It might be helpful to copy the code into an online tool like [CodePen](https://codepen.io/), [jsFiddle](https://jsfiddle.net/), or [Glitch](https://glitch.com/) to test your work.
> If you get stuck, you can [ask us for help](#assessment_or_further_help).

## Math 1

Let's start out by testing your knowledge of basic math operators.

> You will have to create four numeric values, adding two together, subtracting one from another, then multiplying the results.

Reviewer: ```suggestion
You will create four numeric values, add two together, subtract one from another, then multiply the results.
```

Finally, we need to write a check that proves that this value is an even number.

Try updating the live code below to recreate the finished example by following these steps:

> 1. Create four variables with sensible names that contain numbers.

Author: ```suggestion
1. Create four variables that contain numbers. Call the variables something sensible.
```

> 1. Create four variables with sensible names that contain numbers.

Reviewer: This is ambiguous and can mean "the names contain numbers". Original reads better.

Author: Good point, have reverted

2. Add the first two variables together and store the result in another variable.
3. Subtract the fourth variable from the third and store the result in another variable.
4. Multiply the results from steps **2** and **3** and store the result in a variable called `finalResult`.
5. Check whether `finalResult` is an even number and store the result in a variable called `evenOddResult`.

> To pass this test, you should have a result of `48`.

Reviewer: The "result" here is referring to `finalResult`, but that's slightly ambiguous. What about inlining it in step 4, or explicitly mention the values for both `finalResult` and `evenOddResult`?

Author: Added some changes in 9e139acb1e20673a6ce3b5b4b78622c3dfa92087

## Math 2

In the second task, you are provided with two calculations with the results stored in the variables `result` and `result2`.

> You need to take the calculations and change their formatting so it has two decimal places.

Reviewer: ```suggestion
You need to take the calculations, multiply them, and format the result to two decimal places.
```

...I think?
